.circleci/config.yml

```diff
@@ -198,19 +198,28 @@ jobs:
       - run: *run-tests
 
 
-  cpp-linux:
+  cpp-gcc:
+    parameters:
+      gcc-version:
+        type: string
+
     docker:
-      - image: cimg/python:3.9
+      - image: gcc:<< parameters.gcc-version >>
 
-    steps: &unix-cpp-tests
+    steps:
       - checkout
       - run:
+          name: install python requirements
+          command: |
+            apt update
+            apt install python3-pip python3-venv -y
```

Comment:
Perhaps worth specifying python3.11 here?

Reply:
It depends on the docker image we're using. This way it'll inherit what's already there (or so I understand).

Reply:
Right, this is only the C++ tests. All good then.